src/components/App.js

```diff
@@ -1,17 +1,163 @@
+//Application ID: cc90edfa
+//Application Key: 3ccb0e8ad4f72be21ef79e0df985a0e
+//Example: https://api.edamam.com/search?q=chicken&app_id=cc90edfa&app_key=3ccb0e8ad4f72be21ef79e0df985a0e
+
+
 import React from 'react';
+import Header from './Header.js'
+import IngredientAdd from './IngredientAdd.js'
+import Fridge from "./Fridge.js"
+import SearchRecipes from "./SearchRecipes.js"
+import Loading from './Loading.js'
 
 import '../styles/components/app.scss';
 
 class App extends React.Component {
 
   constructor(){
     super();
+
+    this.state = {
+
+      display: 'fridge', // 'fridge' or 'recipes' or 'favourites'
+
+      stock: [],
+
+      activeIngredients: [],
+
+      searchResults: [],
+
+      loading: false,
+
+      favourites: [],
+
+      page: 0
+
+    }
+
+    this.retrieveItem = this.retrieveItem.bind(this)
+    this.removeItem = this.removeItem.bind(this)
+    this.retrievePage = this.retrievePage.bind(this)
+    this.fetchRecipes = this.fetchRecipes.bind(this)
+    this.changeDisplay = this.changeDisplay.bind(this)
   }
 
+
+
+componentDidMount(){
+  const currentStock = window.localStorage.getItem('stock');
+  const stockArray = currentStock ? JSON.parse(currentStock) : [];
+  const currentFaves = window.localStorage.getItem('favourites')
+  const favesArray = currentFaves ? JSON.parse(currentFaves) : [];
+  this.setState({
```

Comment:
minor point, if you rename `stockArray` to `stock` and `favesArray` to `faves` you can shorten setState call to
```
this.setState({stock, favourites});